sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -553,6 +587,9 @@
       abstract Builder<K, V> setValueDeserializerProvider(
           DeserializerProvider deserializerProvider);
 
+      abstract Builder<K, V> setCheckStopReadingFn(
```

Comment:
Wasn't this functionality added in #13710 ?

Reply:
Yeah, I opened this PR prior to mergeing #13710. I'll rebase them. Thanks for catching it : )